tests/ethereum/main.rs

```diff
@@ -192,23 +194,18 @@ fn run_test_unit(path: &Path, unit: &TestUnit) {
                     // This is a good reference for a minimal state/DB commitment logic for
                     // PEVM/REVM to meet the Ethereum specs throughout the eras.
                     for (address, account) in state {
-                        if !account.is_touched() {
-                            continue;
-                        }
-                        if account.is_selfdestructed()
-                            || (account.is_empty()
-                                && spec_id.is_enabled_in(SpecId::SPURIOUS_DRAGON))
-                        {
+                        if let Some(account) = account {
+                            if account.is_empty() && spec_id.is_enabled_in(SpecId::SPURIOUS_DRAGON) {
```

Comment:
Looking at this, I suspect it's better to add this check to `transform_output`'s filtering.
Then the state transition logic for library users would be so clean: Extend on `Some` and delete on `None`, no extra checks, no need to worry about eras, etc.

After that, I suppose we can promote `transform_output` to `post_process_results` now (not just about formatting but now also with EVM-specific logic).